cmd/main.go

```diff
@@ -10,6 +10,9 @@ import (
 	"github.com/jtblin/kube2iam/iptables"
 	"github.com/jtblin/kube2iam/server"
 	"github.com/jtblin/kube2iam/version"
+	"os"
```

Comment:
Can you move the ones below with the rest of the standard library i.e. "strings"?

Reply:
👍